### ISSUE TITLE
lsp: Reexports (for dependencies) require the file containing the reexports to be open to resolve imports

### ISSUE THREAD

**Issue by somebody1234** [denoland/deno#10125]
**Describe the bug**

**To Reproduce**
1. Create file containing the following (any module can be substituted in):
```ts
export * from 'https://deno.land/x/discordeno@10.5.0/mod.ts';
```
2. Import something reexported from this module
3. Close the file containing the reexports
4. Edit anything in the file containing the import to trigger an update

**Expected behavior**
Everything resolves correctly

**Screenshots**
-

**Versions**

vscode: code-server 1.53.2
deno: 1.8.1 (release, x86_64-unknown-linux-gnu)
extension: 3.2.0
note that code-server is running as a service (and as a side effect, as root (ew, i know)). unsure whether this is related to the issue
if it helps, code lens > references and code lens > references all functions are off, unstable is on

**Comment by kitsonk** [comment]
I have certainly encountered this issue, but it is hard to understand when it is actually occurring, as I have tried to recreate it specifically, and I can't seem to.  I have opened #10253 to hopefully provide more diagnostic information when it occurs and will be running that locally trying to re-create it.

It certainly has to do with re-exports, but it clearly isn't every situation, and as other have stated, restarting usually clears the issue (which is what makes it hard to reproduce as well).
